src/ui/reducers/app.ts

```diff
@@ -64,13 +64,6 @@ const appSlice = createSlice({
     setMouseTargetsLoading(state, action: PayloadAction<boolean>) {
       state.mouseTargetsLoading = action.payload;
     },
-    durationSeen(state, action: PayloadAction<number>) {
-      // If this is the longest duration we have seen yet, we should update the
-      // store
-      if (action.payload > state.recordingDuration) {
-        state.recordingDuration = action.payload;
-      }
-    },
```

Comment:
We should be using `getEndpoint` for this. I have not seen anything not work with this deleted, but lmk if there's something else I should check

Reply:
I do see several references to `recordingDuration`, particularly around playback.  Looks like it also gets passed via context into some of the `replay-next` functionality.  Can we swap this out with the better API right now?

Reply:
Yeah I'll see what I can do in a bit!

Reply:
It seems like [this line](https://github.com/replayio/devtools/blob/main/src/ui/actions/timeline.ts#L115) is doing exactly what I was suggesting?

Reply:
Hmm. Seems like, it, yeah?  (Also, putting on my Redux hat: I _hate_ that we have an action named `setTimelineState`. That is literally the exact opposite mentality of how we want people to _think_ about using Redux :(  )

Reply:
Fair!